modules/test/playwright/tests/configuration-admin-web/instanceSettings.spec.ts

```diff
@@ -88,8 +99,69 @@ test('LPD-35562 Enter reserved screen name', async ({page}) => {
 
 	await page.getByLabel('Last Name').fill(lastName);
 
-	await instanceSettingsPage.saveAndWaitForAlert(
-		'Error:The screen name you requested is reserved.',
-		{autoClose: false, type: 'danger'}
-	);
+	await instanceSettingsPage.saveAndWaitForAlert({
+		autoClose: false,
+		text: 'Error:The screen name you requested is reserved.',
+		type: 'danger',
+	});
+});
+
+test('LPD-38043 Assert that a configuration at the site scope can override a falsy configuration at the instance scope', async ({
+	accessibilityMenuPage,
+	instanceSettingsPage,
+	page,
+	site,
+	siteSettingsPage,
+}) => {
+	if (accessibilityMenuPage.isAccessibilityMenuAttached()) {
+		await test.step('Check that the instance scoped accessibility menu configuration is disabled', async () => {
+			await instanceSettingsPage.goToInstanceSetting(
+				'Accessibility',
+				'Accessibility Menu'
+			);
+
+			await accessibilityMenuPage.enableAccessibilityMenuCheckbox.uncheck();
+
+			await instanceSettingsPage.saveAndWaitForAlert();
+		});
+	}
+
+	await test.step('Check that the accessibility menu is not accessible in the site scope', async () => {
+		await siteSettingsPage.goToSiteSetting(
+			'Accessibility',
+			'Accessibility Menu',
+			site.friendlyUrlPath
+		);
+
+		await page.waitForLoadState();
+
+		await expect(
+			accessibilityMenuPage.openAccessibilityMenuButton
+		).not.toBeAttached();
+	});
+
+	await test.step('Enable the site accessibility configuration', async () => {
+		await page.waitForTimeout(1000);
```

Comment:
Hey @thiago-buarqque expect.toPass is actually better than a timeout since it can happen much faster and also handle situations when things are slower. 

A hydration issue like this is actually a bug. So the best thing for us to do is to fix the issue in the component making sure that it is disabled until the click handlers are added. This could be a bit more work right now, so if you don't want to look into that, we can simply use the expect.toPass pattern.

The one thing I'd recommend is that you do an assertion before the expect to pass, so that you know the checkbox had the expected state before trying to change it. Otherwise, the checkbox may have had an unexpected state and the expect.toPass will just toggle it twice until it reaches the desired state to pass.

Reply:
@ethib137  

> A hydration issue like this is actually a bug. So the best thing for us to do is to...

I will not work on this now, since it's related to the component (and we have similar issues in other components)

> The one thing I'd recommend is that you do an assertion before the expect to pass, so that you know the checkbox had the expected state before trying to change it

That checkbox will always be unchecked. For this test a new site is created and the default value for that configuration is false.

Reply:
@ethib137 should I create a ticket to deal with the hydration problem?